Upstatement/ups-public-library
# Index page search

## Changes

Closes #15 

- Adds endpoints for entries, collections, and series to make the data available for search with Fuse
- Adds search bar and logic to display search results on index page

## Steps to test

- Open up the index page and search for something. Make sure relevant results show up
- Make sure tabbing through search results and using arrow keys to cycle through focusable elements works as expected

## Notes

I reverted the `entries` array for collections and series back to titles instead of slugs in order to make my life easier for displaying search results (it was easier to slugify rather than unslugify)
